helm-chart/binderhub/templates/deployment.yaml

```diff
@@ -70,8 +65,7 @@ spec:
         volumeMounts:
           - mountPath: /etc/binderhub/config/
             name: config
-          - mountPath: /etc/binderhub/secret/
-            name: secret-config
+            readOnly: true
```

Comment:
Just mount one file now into /etc/binderhub/config. I made it readOnly as well because why not, it improves performance a bit for k8s.